partitura/io/__init__.py

```diff
@@ -32,6 +34,32 @@ class NotSupportedFormatError(Exception):
     pass
 
 
+def is_url(input):
+    try:
+        result = urlparse(input)
+        return all([result.scheme, result.netloc])
+    except ValueError:
+        return False
+
+
+def download_file(url):
+    # Send a GET request to the URL
+    with urllib.request.urlopen(url) as response:
+        data = response.read()
+
+    # Extract the file extension from the URL
+    extension = os.path.splitext(url)[-1]
+
+    # Create a temporary file
+    temp_file = tempfile.NamedTemporaryFile(delete=False, suffix=extension)
```

Comment:
Won't it be the case that this file is never deleted, and keep accumulating on the computer?

Reply:
Yes, I agree.  Probably, I need to find a better solution and debug. The load score file needs access to the file to check the extension but maybe when `delete=True` the file is not deleted immediately when loaded. I will push a change soon.

Reply:
Turns out that for python version 3.8 the temporary file options are not the same as for newer versions of python therefore I had to add more edits to ensure the deletion of tmp files immediately after the end of the function.